shoop/xtheme/views/editor.py

```diff
@@ -83,10 +88,12 @@ def _populate_vars(self):
         theme = get_theme_by_identifier(self.request.GET["theme"])
         if not theme:
             raise Problem(_("Unable to determine current theme."))
+        view_name = self.request.GET["view"]
+        global_type = self.request.GET.get("global_type", None)
         self.view_config = ViewConfig(
             theme=theme,
-            view_name=self.request.GET["view"],
-            draft=True
+            view_name=(XTHEME_GLOBAL_VIEW_NAME if global_type else view_name),
```

Comment:
No need to override the view_name here since ViewConfig init does that for you. Also missing the global_type parameter you need to pass for the ViewConfig.